### PR TITLE
Irwin coord mapper

### PR DESCRIPTION
**Pull request by iljungr**
- Implemented CoordMapper with unit test. Space usage is not optimized in current implementation, but should be fine for viruses.
- Made muscle quite by default.

### CODE REVIEW

**Comment by iljungr** [comment]
https://github.com/broadinstitute/viral-ngs/issues/99

File: test/unit/test_interhost.py [hunk 1]
```diff
@@ -13,17 +13,37 @@ def test_help_parser_for_each_command(self):
...
+            for chrom, mappedPos in result :
+                self.assertEqual(chrom, toChrom)
+            self.assertEqual(expected,
+                             [mappedPos for chrom, mappedPos in result])
+
+        # Check that IndexError is raised when past ends of other sequence
+        for pos in [-1, 0, 1, 2, 22, 23, 24] :
+            try :
+                cm.mapAtoB('chr1', pos)
+                self.assertTrue(False) # Should never get here
```
Comment (dpark01) [inline test/unit/test_interhost.py]: This block can be replaced with a self.assertRaises

**Comment by dpark01** [comment]
This is awesome. I'll stare at the edge cases more carefully later today when I get a chance, but my guess is that this is spot on. Nice thing is that we can refactor the implementation later on for better space utilization, but having the test coverage in place makes that far easier.

**Comment by iljungr** [comment]
There is no need to refactor in order to get better space utilization. Just define a class for storing the information that implements append, __len__, __getitem__, and __setitem__, that efficiently stores the elements in a list if there are many runs of consecutive integers. Then change the one line "map = []" to "map = MyNewClass()".

I'll put comments in the code to that effect.

File: test/unit/test_interhost.py (hunk 1 above)
Comment (iljungr) [inline test/unit/test_interhost.py]: Thanks. Done.
